src/findex/sqlite.ts

```diff
@@ -10,33 +10,38 @@ import {
 
 /**
  * @param db the SQLite3 connection
+ * @param init create indexes-tables
  * @returns the callbacks
  */
-export function callbacksExamplesBetterSqlite3(db: Database): {
+export function callbacksExamplesBetterSqlite3(
+  db: Database,
+  init?: boolean,
```

Comment:
Do not use `?:` in TypeScript, it's much clearer to use classic default parameter `init: boolean = true`
```suggestion
  init: boolean = true,
```

Reply:
This function is already use in multiple place, the default should preserve the previous behavious (so the default should be `false`)

Reply:
I think it may be better to not have this parameter and use `CREATE TABLE IF NOT EXISTS` (I think it's working in Sqlite)